.pylintrc

```diff
@@ -81,7 +82,7 @@ limit-inference-results=100
 
 # List of plugins (as comma separated values of python module names) to load,
 # usually to register additional checkers.
-load-plugins=pylint_django
+load-plugins=pylint_django, pylint_pytest
```

Comment:
This plugin doesn't seem to be related to models. Why did you list it here?

Reply:
I'll remove this

Reply:
Marking as resolved.